docs/examples/pytorch/pytorch-resnet50.rst

```diff
@@ -20,7 +20,7 @@ Requirements
 .. role:: bash(code)
    :language: bash
 
-- `APEx <https://www.github.com/nvidia/apex>`_
+- `APEx <https://www.github.com/nvidia/apex>`_ - optional, required for fp16 mode or distirbuted (mutli GPU) operation
```

Comment:
"distributed"

Reply:
Sorry, my review from a few hours ago had been left in a pending state.  See below.

Reply:
Done